### PR TITLE
Make pure pursuit use trapezoidal profiling

### PR DESCRIPTION
**Pull request by RinHa**

### CODE REVIEW

**Comment by shadaj** [comment]
@RinHa looks like there is a failure because the robot is ramping up to 100% speed instead of directly going there. Maybe this can be fixed by allowing a super large acceleration in that test?

**Comment by shadaj** [comment]
Ping @Paxelord @RinHa we really should get this merged.

**Review by PhilipAxelrod — APPROVED** [review]
Lgtm

**Review by shadaj — APPROVED** [review]
:shipit:

File: commons/src/test/scala/com/lynbrookrobotics/potassium/commons/drivetrain/PurePursuitTasksTest.scala [hunk 1]
```diff
@@ -83,40 +83,41 @@ class PurePursuitTasksTest extends FunSuite {
...
-    task.init()
-
-    ticker(Milliseconds(5))
-    ticker(Milliseconds(5))
-    ticker(Milliseconds(5))
-    ticker(Milliseconds(5))
-
-    assert(!task.isRunning)
-  }
+//  test("If target is initally within tolerance, stop immediately") {
```
Comment (shadaj) [inline commons/src/test/scala/com/lynbrookrobotics/potassium/commons/drivetrain/PurePursuitTasksTest.scala:86]: @Paxelord @RinHa post-merge: why was this test disabled? does the robot drive when it is within the tolerance?

**Pull request merged**